dev-php/ca-bundle/ca-bundle-1.0.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Find a path to the system CA bundle, and fallback to the Mozilla CA bundle"
+HOMEPAGE="https://github.com/composer/ca-bundle"
+
```

Comment:
.skel uses blanks to make the comments more readable. I think the author meant to use double blank lines where single blank line is intended in ebuild but somehow failed that around EAPI/inherit. Here's a reference: https://devmanual.gentoo.org/quickstart/index.html